notebooks/fig1.ipynb

```diff
@@ -0,0 +1,1561 @@
+    "bot_phyla = taxonomy_df[\"taxonomy_1\"].value_counts()[6:].index.tolist()\n",
+    "top_5_phyla.remove(\"Unspecified\")\n",
+    "bot_phyla.append(\"Unspecified\")\n",
+    "phylum_assignments = [\"Other/Unspecified\" if not x in top_5_phyla else x for x in taxonomy_df[\"taxonomy_1\"]]\n",
+    "top_5_phyla_df = pd.DataFrame.from_dict(dict(zip(taxonomy_df.index.tolist(), phylum_assignments)), orient=\"index\")\n",
+    "top_5_phyla_df.columns = [\"phylum_assignment\"]\n",
+    "top_5_phyla_df.index.name = \"featureid\"\n",
+    "top_5_phyla_df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will save this file as a `tsv` file that QIIME2 can read."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Metadata(top_5_phyla_df).save(\"fig1/output/EMP_top_5_phyla.tsv\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Filtering Feature Table\n",
+    "\n",
+    "The original EMP data includes samples from both sterile water blanks and mock communities. We will remove samples of both these types."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[32mSaved FeatureTable[Frequency] to: fig1/output/EMP_tbl_filt.qza\u001b[0m\n"
+     ]
+    }
+   ],
+   "source": [
+    "!qiime feature-table filter-samples \\\n",
+    "    --i-table fig1/output/EMP_tbl.qza \\\n",
+    "    --m-metadata-file fig1/input/EMP_metadata.tsv \\\n",
+    "    --p-where \"[empo_2] NOT IN ('Positive', 'Negative')\" \\\n",
+    "    --o-filtered-table fig1/output/EMP_tbl_filt.qza"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Additionally, we will filter the feature table by the phylogeny to make sure all features present in the feature table are present in the phylogeny as well."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[32mSaved FeatureTable[Frequency] to: fig1/output/EMP_tbl_filt.qza\u001b[0m\n"
+     ]
+    }
+   ],
+   "source": [
+    "!qiime phylogeny filter-table \\\n",
+    "    --i-table fig1/output/EMP_tbl_filt.qza \\\n",
+    "    --i-tree fig1/output/EMP_tree.qza \\\n",
+    "    --o-filtered-table fig1/output/EMP_tbl_filt.qza"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Running EMPress\n",
+    "\n",
+    "Finally, we will generate an EMPire plot combining an interactive tree visualization with a PCoA. The PCoA is very computationally intensive to calculate so we have omitted it here. The code used to generate the unweighted UniFrac distance matrix and PCoA are shown below (not executed)."
```

Comment:
IIRC the pcoa file should be fairly small (specially if generated using fast PCoA), any chance we can include that in this repo?